pwnlib/tubes/tube.py

```diff
@@ -44,8 +40,35 @@ def __init__(self, timeout = default, level = None, *a, **kw):
             self.setLevel(level)
 
         self.buffer = Buffer(*a, **kw)
+        self._newline = None
         atexit.register(self.close)
 
+    @property
+    def newline(self):
+        b'''Character sent with methods like sendline() or used for recvline().
```

Comment:
I'm so sorry for misleading you earlier
```suggestion
        r'''Character sent with methods like sendline() or used for recvline().
```